README.md

```diff
@@ -1,19 +1,471 @@
 # LiveviewChat
 
-To start your Phoenix server:
+[![Elixir CI](https://github.com/dwyl/phoenix-liveview-chat-example/actions/workflows/cy.yml/badge.svg)](https://github.com/dwyl/phoenix-liveview-chat-example/actions/workflows/cy.yml)
 
```

Comment:
Adding hits badge to track how many people view this. 📈 
```suggestion
  [![HitCount](http://hits.dwyl.com/dwyl/phoenix-liveview-chat-example.svg?style=flat-square)](http://hits.dwyl.com/dwyl/phoenix-liveview-chat-example)
  
```
Hope people _appreciate_ your effort! 🤞